esmtools/carbon.py

```diff
@@ -125,3 +128,129 @@ def potential_pco2(t_insitu, pco2_insitu):
     t_sfc = t_insitu.isel(depth=0)
     pco2_potential = pco2_insitu * (1 + 0.0423 * (t_sfc - t_insitu))
     return pco2_potential
+
+
+def spco2_sensitivity(ds):
+    """Generate sensitivities in spco2 for changes in other variables.
+
+    * Lovenduski, Nicole S., Nicolas Gruber, Scott C. Doney, and Ivan D. Lima.
+        “Enhanced CO2 Outgassing in the Southern Ocean from a Positive Phase of
+        the Southern Annular Mode.” Global Biogeochemical Cycles 21, no. 2
+        (2007). https://doi.org/10/fpv2wt.
+    * Gruber and Sarmiento, 2005
+
+    Args:
+        ds (xr.Dataset): containing cmorized variables:
+                            spco2 [ppm]: pCO2,ocean at ocean surface
+                            talkos[mmol m-3]: Alkalinity at ocean surface
+                            dissicos[mmol m-3]: DIC at ocean surface
```

Comment:
Where are you finding the cmorized names? I've been googling and can't find the actual variable names. I'm just finding cmorized units and variable IDs.

Reply:
do you mean the longnames (like pCO2,ocean at ocean surface)? I just took them for the MPI cmorized output.

Reply:
I meant `dissicos` and stuff. We use `DIC` at NCAR and `ALK` for instance. I believe you it's cmorized, I just couldn't find a table with the variable names.